README.md

```diff
@@ -64,6 +64,9 @@ Below listed feature gates need to be enabled in kubernetes v1.12.1
 --feature-gates=VolumeSnapshotDataSource=true,CSINodeInfo=true,CSIDriverRegistry=true
 ```
 
+>NOTE: You can skip seperate installation of kubernetes cluster, GD2 Cluster and CSI deployment if you directly use [GCS](https://github.com/gluster/gcs) installation methods, it should bring your deployment in one shot.
+Refer [GCS deployment guide](https://github.com/gluster/gcs/tree/master/deploy) for more details.
```

Comment:
can we have a link to the README file instead of pointing to the deploy directory?
`https://github.com/gluster/gcs/blob/master/deploy/README.md`

Reply:
hmm.. By default deploy directory list README. but I can change it.